misc/snapshotter/Dockerfile

```diff
@@ -1,31 +1,41 @@
 FROM alpine:3.17.0 AS sourcer
 
 ARG NYDUS_VER=v2.1.5
-
 RUN apk add --no-cache curl
 RUN apk add --no-cache --upgrade grep
-RUN wget https://github.com/dragonflyoss/nydus/releases/download/$NYDUS_VER/nydus-static-$NYDUS_VER-linux-amd64.tgz && \
+RUN curl -OL https://github.com/dragonflyoss/nydus/releases/download/$NYDUS_VER/nydus-static-$NYDUS_VER-linux-amd64.tgz && \
     echo $NYDUS_VER > /.nydus_version && \
     tar xzf nydus-static-$NYDUS_VER-linux-amd64.tgz && \
-    rm nydus-static-$NYDUS_VER-linux-amd64.tgz
-RUN mv nydus-static/* /
+    rm nydus-static-$NYDUS_VER-linux-amd64.tgz && \
+    mv nydus-static/* / && \
+    curl -OL https://github.com/liubin/toml-cli/releases/download/v0.0.7/toml-cli-v0.0.7-linux-amd64.tgz && \
+    tar xzf toml-cli-v0.0.7-linux-amd64.tgz && \
+    mv toml-cli/toml / && \
+    rm toml-cli-v0.0.7-linux-amd64.tgz
 
 FROM alpine:3.17.0
+ARG DESTINATION=/opt/nydus-artifacts
+ARG NODE_DESTINATION=${DESTINATION}/opt/nydus
```

Comment:
`NODE_DESTINATION=/opt/nydus-artifacts/opt/nydus` seems a strange path.

Reply:
This is only a path inside the container, then the daemonset will be able to copy everything from /opt/nydus-artifacts/* to the host (usually in /).

This will lead to /opt/nydus on the host side, which is good.

Reply:
Yes, as @fidencio said, we would copy everything from  `/opt/nydus-artifacts/*` to the host `/` in the function `install_snapshotter`.